SEED-platform/seed
# 1879 profile id v21

Note that this is based off of master (called 2.5.2-patch0).

#### Any background context you want to provide
The listing of properties between version 2 and 2.1 of the API differed in the sense that there was not a way to remove fields in the 2.1 results using the profile_id setup.

#### What's this PR do?
* Adds the profile_id to the properties list for version 2.1 API
* Adds new helper method for extracting the show_columns from a profile_id

#### How should this be manually tested?
* Import data
* Create a profile on the properties list
* Go to the URL for the v2.1 API call and include profile_id in query param  
  `{{SEEDURL}}/api/v2.1/properties/?organization_id={{organization_id}}&profile_id={{profile_id}}
* Make sure that the state data are only the fields in the profile_id.

#### What are the relevant tickets?
#1879 

#### Screenshots (if appropriate)
n/a
